airflow/lineage/backend/atlas/__init__.py

```diff
@@ -35,8 +36,9 @@
 
 
 class AtlasBackend(LineageBackend):
+    """Send lineage metadata to Atlas backend."""
     @staticmethod
-    def send_lineage(operator, inlets, outlets, context):
+    def send_lineage(operator, inlets, outlets, context): # pylint: disable=signature-differs
```

Comment:
In my opinion this warning should get fixed. 

If that's because of the `send_lineage` [here](https://github.com/apache/airflow/blob/66a594e13b969dcf21b58363d72b3b7e8d0d8f4a/airflow/lineage/backend/__init__.py#L23-L24) I think we should make the kwargs required args (remove `=None`). And then remove the static modifier from here. Because I cannot find any static usage nor do I see the arguments being used optionally.

Or do I miss something?

Reply:
I was not sure if this was in the scope of the ticket. I agree with you in making the kwargs required. Currently we have only one backend implementation for the lineage (Atlas).